.github/workflows/test-all-links.yml

```diff
@@ -0,0 +1,12 @@
+name: Check Markdown links
+on: 
+  schedule:
+    - cron: '* */24 * * *'
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.7
+        with:
+          folder-path: "main"
```

Comment:
Is this happy without a final linefeed? It's usually safer to include it.

Reply:
Oh good point, I will include